### DISCUSSION TITLE
recreate auth.users table

### DISCUSSION THREAD

**Discussion by Tahmasb** [supabase/supabase#11682] [Questions]

**Comment by GaryAustin1** [comment]
There are a bunch of migration files that have been added over time to build the auth schema...
https://github.com/supabase/gotrue/blob/master/migrations

With the new postgres user without super user privileges, I'm not even sure you could successfully recreate the auth table with the correct grants,  although it got deleted some how.

You might be better off migrating your data from this damaged instance to a new one.

**Reply by GaryAustin1** [reply]
I have no idea how the code is setup and if it will even run in the SQL editor because of privileges.  It does look like you have to fill in the auth schema thru out the code for it to work.  PLUS just the first one is not enough.  The auth.users table is modified several times in the files that follow over time.

Unless you know what you are doing, you are better off pulling your public schema out and uploading it to a new instance.  I think you are asking for trouble trying to rebuild the auth.users especially if you miss something that is needed down the road.

https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects
